docs/src/utils/getStage.ts

```diff
@@ -0,0 +1,9 @@
+export const getStage = () => {
```

Comment:
Just a tiny TS nit -- what do you think about explicitly defining return types
```suggestion
type Stage = 'dev' | 'test' | 'prod';

export const getStage = (): Stage => {
```

Reply:
+1 to typing the stages, but it should be an enum instead:
```ts
enum STAGE {
  PROD: 'prod';
  DEV: 'dev';
  LOCAL: 'local';
}
```

Also, let's use `local` for our local environments rather than `test`?

Reply:
The reason why I use `test` is because we may test in either local or personal account